src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue28051.cs

```diff
@@ -0,0 +1,26 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue28051 : _IssuesUITest
+{
+	public Issue28051(TestDevice testDevice) : base(testDevice) { }
+
+	public override string Issue => "ObjectDisposedException Occurs During View Recycling When PlatformBehavior Is Attached on Android";
+
+	const string Refresh = "Refresh";
+
+	[Test]
+	[Category(UITestCategories.Performance)]
+	public void RefreshItemsShouldNotCrash()
```

Comment:
The test is failing on Android with a crash:
`The app was expected to be running still, investigate as possible crash`

Could you review it?

Reply:
@jsuarezruiz  Please rerun the CI to resolve this failure. This PR was previously targeting main, but the corresponding test fix has now been moved to the inflight/current branch, and this PR is also updated to target inflight now.